### PR TITLE
Fix keymap reviung41 via

### PR DESCRIPTION
**Pull request by gtips**
## Description
Fixed an issue where ADJUST MO(_ADJUST) did not work.
keyboards/reviung/reviung41/keymaps/via/keymap.c

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/reviung/reviung41/keymaps/via/keymap.c [hunk 1]
```diff
@@ -52,3 +56,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
   ),
 };
 
+layer_state_t layer_state_set_user(layer_state_t state) {
+  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
```
Comment (waffle87) [inline keyboards/reviung/reviung41/keymaps/via/keymap.c:61]:
```suggestion
```

File: keyboards/reviung/reviung41/keymaps/via/keymap.c [hunk 2]
```diff
@@ -22,28 +22,32 @@ enum layer_names {
...
+#define RAISE  MO(_RAISE)
+#define ADJUST MO(_ADJUST)
+
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
   [_BASE] = LAYOUT(
     KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,      KC_T,               KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,     KC_BSPC,
     KC_LCTL,  KC_A,     KC_S,     KC_D,     KC_F,      KC_G,               KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_QUOT,
     KC_LSFT,  KC_Z,     KC_X,     KC_C,     KC_V,      KC_B,               KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  RSFT_T(KC_ENT),
-                                            KC_LGUI,   MO(1),    KC_SPC,   MO(2),    LT(3, KC_RALT)
+                                            KC_LALT,   LOWER,    KC_SPC,   RAISE,    KC_RGUI
```
Comment (waffle87) [inline keyboards/reviung/reviung41/keymaps/via/keymap.c:34]:
```suggestion
                                            KC_LALT,   TL_LOWR,    KC_SPC,   TL_UPPR,    KC_RGUI
```

File: keyboards/reviung/reviung41/keymaps/via/keymap.c [hunk 3]
```diff
@@ -22,28 +22,32 @@ enum layer_names {
     _ADJUST
 };
 
+#define LOWER  MO(_LOWER)
+#define RAISE  MO(_RAISE)
+#define ADJUST MO(_ADJUST)
+
```
Comment (waffle87) [inline keyboards/reviung/reviung41/keymaps/via/keymap.c:28]:
```suggestion
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/reviung/reviung41/keymaps/via/keymap.c (hunk 3 above)
Comment (gtips) [reply keyboards/reviung/reviung41/keymaps/via/keymap.c:28]: I modified the code and checked that it works with the keyboard, but the _ADJUST layer doesn't work.
I tried adding "TRI_LAYER_ENABLE = yes" to the rules.mk file, but it still doesn't work.
Where should I check?

Comment (drashna) [reply keyboards/reviung/reviung41/keymaps/via/keymap.c:28]: via enables tri layer support anyways.  

Also, if you have the layer_state_set_user function still, that will override the keycodes, and prevent them from working.

Comment (gtips) [reply keyboards/reviung/reviung41/keymaps/via/keymap.c:28]: Sorry, It seems that this fix was a misunderstand on my part.
I will close this pull request.
Thanks.

**Comment by gtips** [comment]
Sorry, It seems that this fix was a misunderstand on my part.
I will close this pull request.
Thanks.

**Pull request closed without merging**